manim/mobject/mobject.py

```diff
@@ -200,6 +200,13 @@ def construct(self):
                         c.animate(rate_func=there_and_back).shift(RIGHT),
                     )
 
+        .. warning::
+
+            `s.animate(run_time=2).rotate(PI / 2)`` will only
+             interpolate between the current state of the mobject, and the state of the mobject after applying the commands after .animate to it.
```

Comment:
```suggestion
             will interpolate the :class:`~.Mobject` between its points prior to ``.animate`` and its points after applying ``.animate`` to it. This may result in unexpected behavior when attempting to interpolate along paths, or rotations.
```